src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -187,14 +183,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 
       <!-- This is the list of inputs that will be used for generating the manifest used during development. -->
     <ItemGroup>
-      <Content
+      <ContentWithTargetPath
         Include="$(_GeneratedStaticWebAssetsDevelopmentManifest)"
-        Condition="'@(_ExternalStaticWebAsset->Count())' != '0'"
-        Link="$(TargetName).StaticWebAssets.xml">
+        Condition="'@(_ExternalStaticWebAsset->Count())' != '0'">
 
+        <TargetPath>$(TargetName).StaticWebAssets.xml</TargetPath>
         <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         <CopyToPublishDirectory>Never</CopyToPublishDirectory>
```

Comment:
Does this need to set the `ExcludeForSingleFile` flag?

Reply:
This file doesn’t get published, so it doesn’t need it.